Resources/Prototypes/Entities/Objects/Consumable/Food/burger.yml

```diff
@@ -254,12 +254,14 @@
   - type: SolutionContainerManager
     solutions:
       food:
-        maxVol: 33
+        maxVol: 45
```

Comment:
math checks out for the burgers, gains a few nutriments over the source ingredients